modules/administration-guide/partials/assembly_configuring-prod-cli-to-use-a-backup-server-using-the-configuration-object.adoc

```diff
@@ -0,0 +1,18 @@
+:parent-context-of-configuring-prod-cli-to-use-a-backup-server-using-the-configuraiton-object: {context}
```

Comment:
I'm confused here. Is `prod-cli` the name of the product or should it be a variable?

Reply:
@bburt-rh, apologies. This file is a leftover that should have been deleted.